File: youqugou-interface/src/main/java/cn/itcast/core/service/SpecificationAuditService.java
package cn.itcast.core.service;

import cn.itcast.core.entity.PageResult;
import cn.itcast.core.pojo.specification.Specification;

public interface SpecificationAuditService {
    PageResult searchSpecificationAudit(Integer pageNum, Integer pageSize, Specification specification);

    void updateStatus(Long[] ids, String status);
}

File: youqugou-web-manager/src/main/webapp/js/service/specificationAuditService.js
app.service("specificationAuditService", function ($http) {
    this.searchSpecification = function (currentPage, itemsPerPage, searchEntity) {
        return $http.post("../specificationAudit/searchSpecificationAudit.do?pageNum=" + currentPage
            + "&pageSize=" + itemsPerPage, searchEntity);
    }


    this.deleteSpecificationVo = function (ids) {
        return $http.post("../specification/deleteSpecificationVo.do",ids);
    }

    this.selectAllSpecificationMap = function () {
        return $http.get("../specification/selectAllSpecificationMap.do");
    }
});